packages/vhd-lib/Vhd/VhdDirectory.js

```diff
@@ -162,7 +163,17 @@ exports.VhdDirectory = class VhdDirectory extends VhdAbstract {
     )
 
     const compressed = await this.#compressor.compress(buffer)
-    return this._handler.outputFile(this._getChunkPath(partName), compressed, this._opts)
+
+    // some objectstorage provider like backblaze, can answer a 500/503 routinely
+    // in this case we should retry,  and let their load balancing do its magic
+    // https://www.backblaze.com/b2/docs/calling.html#error_handling
+    return pRetry(async () => this._handler.outputFile(this._getChunkPath(partName), compressed, this._opts), {
+      tries: 5,
```

Comment:
`promise-toolbox` waits by default on second between two tries, is that what you want?

In any case, I think we should explicitly set this value, in case the default change in the future.